### ISSUE TITLE
switch to ipynb in pandoc 2.6, add cross pip-conda py-xx modules 

### ISSUE THREAD

**Issue by kiwi0fruit** [kiwi0fruit/pandoctools#35]
* [x] ~test pandoctools in virtual and conda environments~
* [x] ~does shortcutter search CWD? Nope it doesn't~
* [x] ~pandoctools_bin in ready is bad set~
* [x] ~if stderr affect conda tests then redirect stderr (make sure that it's separate to error code). Nope it doesn't affect tests.~
* [x] ~cygpath check should check error code too~
* [x] ~fix pandoctools documetnation~
  * [x] ~remove knitty<0.5~
  * [x] ~https://pandoc.org/MANUAL.html#creating-jupyter-notebooks-with-pandoc~
  * [x] ~pandoc-filter-arg~
  * [x] ~to pandoctools 2.6 update instructions~
  * [x] ~write that you can have bash in the activated env and Pandoctools would use it~
* [x] ~in pandoctools search for bash with bat ext also (+fix cygpath search)~
* [x] ~test how git-bash-win post install works when git:conda-forge is installed.~
* [x] ~test if activation of env with git-bash-win from system bash overrides bash (bash.bat vs bash): not needed as bash.bat is invisible for bash~
* [x] ~`$(git --exec-path)` was buggy because I ran installation from system git bash and maybe via `call`. TEST and echo path obtained from 'git' exec.~
* [x] ~run `.\git-bash...`~
* [x] ~first run of the batch.bat would echo unnecessary text.~
* [x] ~reproduce program files git error at home, check that leftover files do not prevent reinstall, check that there's really no linking, try creaning `pkgs/git-bash-win*` before installing, `Scripts\` dissappear even now. I guess it's a git-bash post-install bug (test it a lot!). Even when I fix post install then break it again and investigate Scripts dir disappearance.~
  ```python
  print(b''.decode(encoding='cp1251'))
  print(b''.decode(encoding='cp866'))
  ```
  ```bat
  :: debug:
  chcp 65001 > nul
  set "LANG=C.UTF-8"
  ```
* [x] ~[`git-bash.exe --no-needs-console --hide --no-cd --command=post-install.bat`](https://github.com/git-for-windows/build-extra/blob/master/post-install.bat)~
* [x] ~change py-xx descriptions so that it's clear that it's only a mathjax/pandoc/pandoc-crossref conda package installer.~
* [x] ~backup py-bash repo~
*  [x] ~**fix knitty and stitch documetnation (readme, examples, pystitch)**:~
  * [x] ~knitty first arg is filter's to~
  * [x] ~pandoc-filter-arg~
  * [x] ~knotr -> pandoc + knitty~
  * [x] ~write about knotedown and about pandoc 2.6~
  * [x] ~remove `--to-ipynb` from everywhere (README.md, click decorators)~
  * [x] ~add `knitty.self_contained_raw_html_img` to knitty docs~
  * [x] ~pandoc-crossref: download in Knitty README.md~
  * [x] ~`{chunk=None}`~
* [x] ~check that knitty output with raw code is handled py pandoc~
* [x] ~update pandoctools examples~
* [x] ~does really self-contained images work (I doubt pipe after if-fi)? Yep, it does.~
* [x] ~test r kernel spec in metadata when r.ipynb~
* [x] ~test knitty (with pandoctools chain) as tests don't cover new CLI~
* [x] ~knitty doesn't remove --to [FIXED]~
* [x] ~Integrate `knitty.self_contained_raw_html_img` to pandoctools~
* [x] ~fix `--to r.ipynb:markdown+xx` -> `--to r.ipynb+xx`~
* [x] ~remove knotr and fix tests~
* [x] ~upd both py-pandoc-crossref https://github.com/conda-forge/py-pandoc-crossref-feedstock/commits/master~
* [x] ~control https://github.com/conda-forge/pandoc-crossref-feedstock/commits/master~
* [x] ~knitty first arg is filter's to. Simplify latex and dir logic.~
* [x] ~add pandoc-filter-arg to knitty~
* [x] ~move _package.yml to readmes~
* [x] ~check https://github.com/conda-forge/pandoc-crossref-feedstock/commits/master and https://github.com/conda-forge/py-pandoc-crossref-feedstock/commits/master~
* [x] ~Keep it simple. But also keep knotedown as it is now. `input=True` is also OK~
* [x] ~make second positional arg support in knitty friendly to custom classes~
* [x] ~add 2.6 py-pandoc to conda-forge~
* [x] ~upd pandoc-crossref to pandoc 2.6 or add pandoc-crossref `0.3.4.0-2` with `"pandoc>=2.5,<=2.6"` as a half-measure (py-pandoc-crossref `0.3.4.0.2` with `"py-pandoc>=2.5,<2.6.1"`). That's bad but let it be.~
* [x] ~this should fix https://github.com/kiwi0fruit/knitty/issues/16~
* [x] ~move double functions to knitty but set import there empty~
* [x] ~pandoc cannot read read-only files on Windows (needs write atrributes permission) https://github.com/jgm/pandoc/issues/5259 test 2.6~
* [x] ~fork pandoc-attributes to pandoc-attrs.~
* [x] ~test ipynb export in 2.6~
* [x] ~fix where: so that it uses which on unix and can check .bat and .cmd also (`exe_only: bool=True`). (both in pandoctools and in knitty)~
* [x] ~add `pyppdf-replace-mathjax` from pyppdf to pandoctools as python script. Rename Default.py to Default_mathjax.py (`$mathjax`) as putting all python helpers in one file is bad idea. There should be `sub` and `sub_pdf` functions there.~
* [x] ~Temporary add panflute filter that iterates over images and converts them to `pandoc -f json -t html --standalone --self-contained`, regex takes `<figure>...</figure>`, puts inside `` ```{=html}\n...\n``` ``. `pandoctools.no_cell_attachments`  that can read `--self-contained --standalone` it doesn't need special folder!~
* [x] ~remove dependence from pypandoc,  remove CDW pandoc exception. Patch [panflute's which](https://github.com/sergiocorreia/panflute/blob/master/panflute/tools.py): `which=where`~
* [x] ~GutHub now supports Pandoc attractive. Maybe Jupyter supporta too? Or test if other classes are stripped in code blocks in Markdown.~
  ````
  ```{.py .code .cell}
  print(0)
  ```
  ````
* [x] ~remove `--to-ipynb` from everywhere~
* [x] ~remove post-knitty~
* [x] ~set pandoctools version to 2.6~
* [x] ~`pandoc>=2.6` dep directly in pandoctools - needed for writeprotected user profile~
* [x] ~remove pandoc-filter-arg from pandoctools~
* ~pandoctools should fall when anything falls in the profile... It should but I'm not going to implement it.~
* ~Move latex knotr logic to convert so that it would play with two different 'to'.~
* ~weaken pandoc dep in pandoctools~
* ~fix knitty eval option (should work on lang names not kernel names)~
  ``` 
  ---
  eval:
    py: True
    pug: False
    other: True
  ...
  ```
* ~move parts of knitty readme to pystitch~
* ~try https://github.com/necolas/normalize.css/ in pandoctools. Normalize or Reset CSS are usually part of most boilerplates. I think Bootstrap 4 uses a CES reset internally as well.~
* ~make work with any Pandoc version~
* ~wait for https://github.com/jgm/pandoc/issues/5299 fix or https://github.com/jgm/pandoc/issues/5303~
